deeplearning4j-keras/src/main/java/org/deeplearning4j/keras/RecursiveCopier.java

```diff
@@ -0,0 +1,33 @@
+package org.deeplearning4j.keras;
+
+import org.nd4j.linalg.api.ndarray.INDArray;
+
+public class RecursiveCopier {
+
+    private final INDArray output;
+    private final float[] input;
+    private final int[] shape;
```

Comment:
Use databuffers here. (Nd4j.createBuffer()) the input should never assumed to be float.